### PR TITLE
Dataprep

### PR DESCRIPTION
**Pull request by umer0zerve**
pypeline updates

The pipeline consists of code for performing the following functionality:
Preprocessing
Outlier handling
Imputation
Encoding
datetime feature transformation
discretization

### CODE REVIEW

**Review by kreshnaa-raam — COMMENTED** [review]
Lets wait for Jason's review before merge.

**Review by jasonhillary — APPROVED** [review]

**Pull request merged**
